integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/bootstrap/ElasticsearchBootstrapIT.java

```diff
@@ -270,6 +276,49 @@ public void noVersionCheck_versionOverrideOnStart_compatibleVersion() {
 		assertThat( elasticsearchClientSpy.getRequestCount() ).isNotZero();
 	}
 
+	/**
+	 * Check custom settings/mapping can be parsed when version_check.enabled is false,
+	 * meaning the elasticsearch link is only established *after*
+	 * the custom settings/mapping are parsed.
+	 */
+	@Test
+	@TestForIssue(jiraKey = {"HSEARCH-4435"})
```

Comment:
a minor stuff: usually we don't put the curly brackets around the jira key

Reply:
Right, probably some IDE auto-complete gone wrong. Well, I'll be more careful next time; no need to bother CI with such a change.